app/views/layouts/dashboard.html.haml

```diff
@@ -51,7 +51,7 @@
               = nav_item_link t('dashboard.navigation.skills'),
                               dashboard_skills_url,
                               policy_name: %i[dashboard skill]
-              = nav_item_link t('dashboard.navigation.ideas'),
+              = nav_item_link "#{t('dashboard.navigation.ideas')} #{count_pending_ideas}",
```

Comment:
it can be done without string interpolation. Check this article https://robots.thoughtbot.com/pluralizing-rails-i18n-translations

Reply:
@tsrandrei nope, take a closer look at the article, it has:

```
en:
  notification:
    one: You have 1 notification
    other: You have %{count} notifications
```

```
<%= t("notification", count: current_user.notifications.count) %>
```

So it is possible to define two different translations and the framework will choose single or plural form automatically by the number you have passed to the `count` key.

Reply:
As the original code, I was thinking it was good to have Ideas for both,singular and plural. I will change it and push back :)

Reply:
Should I change also for Applicants? 
https://monosnap.com/file/L45MUoYeSHHg6Pz9cmjifVnGZQUOS6

Reply:
Probably. Also please post the screenshots to the PR description when you'll done with the implementation.